app/retail/templates/shared/head.html

```diff
@@ -49,6 +49,7 @@
 <!-- Grants -->
 <link rel="stylesheet" href="{% static "v2/css/grants/grant.css" %}">
 <link rel="stylesheet" href="{% static "v2/css/grants/new.css" %}">
+<link rel="stylesheet" href="{% static "v2/css/grants/milestones.css" %}">
```

Comment:
@timborden could we move this over to details page cause ideally, it won't appear elsewhere ?

Reply:
Happy to make that change....just want to confirm, I'm moving the css styles from `v2/css/grants/milestones.css` into `v2/css/grants/detail.css` and deleting the `milestones.css` file.

Is that correct?

I agree that there are some styles shared between the `detail.html` and `milestones.html` pages....there are also some styles in `milestones.css` that are for the input form that only displays in `milestones.html`, are we okay having those in `detail.css`?

Reply:
@timborden no no this is good ! I meant can you move `<link rel="stylesheet" href="{% static "v2/css/grants/milestones.css" %}"` into the `milestones.html` so that we don't import it into every file even when it's not needed

Reply:
That makes sense....thanks for clarifying.